stacks-common/src/deps_common/bitcoin/network/message_network.rs

```diff
@@ -23,7 +23,6 @@ use crate::deps_common::bitcoin::network::constants;
 use crate::util;
 
 /// Some simple messages
-
```

Comment:
I think we actually want this whitespace, but the line above should be `//` instead of `///`